spread.yaml

```diff
@@ -3,7 +3,7 @@ project: snapd
 environment:
     GOHOME: /home/gopath
     GOPATH: $GOHOME
-    REUSE_PROJECT: "$(HOST: echo $REUSE_PROJECT)"
+    REUSE_PROJECT: "$(HOST: echo \"$REUSE_PROJECT\")"
```

Comment:
`\"` looks kinda ugly, I think this would work with:
```
REUSE_PROJECT: '$(HOST: echo "$REUSE_PROJECT")'
```

Reply:
I wasn't aware that YAML had both type of strings. If that works +1 for changing it.

Reply:
+1, I'll open a separate PR for that though.

Reply:
Actually, it's here now.